src/components/common/MapLegend/index.tsx

```diff
@@ -74,13 +76,20 @@ export default function MapLegend({ legends, setLegends }: MapLegendProps): JSX.
                 />
               )}
               <Typography
-                fontSize='14px'
+                fontSize='16px'
                 fontWeight={600}
                 width={isMobile ? '100%' : undefined}
                 marginRight={isMobile ? 0 : theme.spacing(4)}
                 paddingLeft={legend.switch ? theme.spacing(1) : 0}
               >
-                {legend.title}
+                {legend.title}{' '}
+                {legend.tooltip && (
```

Comment:
Is the space necessary after the title? `{' '}`

Reply:
We need space, but I removed it and added a margin